### PR TITLE
TASK: Lots of code cleanup

### PR DESCRIPTION
**Pull request by simonschaufi**
- include `$kga = Kimai_Registry::getConfig();` where needed
- include `$database = Kimai_Registry::getDatabase();` where needed

### CODE REVIEW

File: core/kimai.php [hunk 1]
```diff
@@ -19,11 +19,13 @@
...
 // = Smarty (initialize class) =
-// ============================= 
+// =============================
 require_once '../includes/basics.php';
 
+$database = Kimai_Registry::getDatabase();
+
 $view = new Zend_View();
-$view->setBasePath(WEBROOT . '/templates');
+$view->setBasePath(WEBROOT . 'templates');
```
Comment (kevinpapst) [inline core/kimai.php]: Here and for all following occurences:
- the constant should always end with NO trailing slash
- the usages should always incorporate a starting slash

File: core/processor.php [hunk 2]
```diff
@@ -278,7 +280,8 @@
                     $errorMessages['customerGroups'] = $kga['lang']['atLeastOneGroup'];
                 }
 
-                if (!checkGroupedObjectPermission('Customer', $id ? 'edit' : 'add', $oldGroups, $_REQUEST['customerGroups'])) {
+                if (!checkGroupedObjectPermission('Customer', $id ? 'edit' : 'add', $oldGroups,
+                    $_REQUEST['customerGroups'])) {
```
Comment (kevinpapst) [inline core/processor.php:284]: Why the break? what about increasing the line length limit or simply ignore some long lines?
Just for me, this is less readable than before - and before it took only half of my screen...
Same goes for all following places.

File: extensions/ki_expenses/processor.php [hunk 3]
```diff
@@ -217,8 +220,10 @@ function ($sum, $expense) {
                 $kga['lang']['errorMessages']['emptyField'],
                 $kga['lang']['designation']
             );
-        } else if (!is_numeric($_REQUEST['edit_value'])) {
-            $errors['edit_value'] = $kga['lang']['errorMessages']['wrongData'];
+        } else {
+            if (!is_numeric($_REQUEST['edit_value'])) {
```
Comment (kevinpapst) [inline extensions/ki_expenses/processor.php]: why split it off in a separate if?

**Review by kevinpapst — COMMENTED** [review]
To be honest, I stopped after 10 minutes, this is way to large to review it properly. Besides, it looks like automatic code formatting, which is bitchy with so many oen PR - you might create a lot of conflicts ;-)
Are all these code formattings part of the scrutinizer check?

**Comment by simonschaufi** [comment]
>  Are all these code formattings part of the scrutinizer check?

no, i did everything by hand.

File: core/kimai.php (hunk 1 above)
Comment (simonschaufi) [reply core/kimai.php]: done

File: core/processor.php (hunk 2 above)
Comment (simonschaufi) [reply core/processor.php:284]: done

File: extensions/ki_expenses/processor.php (hunk 3 above)
Comment (simonschaufi) [reply extensions/ki_expenses/processor.php]: misunderstanding.

**Pull request merged**